lib/Installation/Partitioner/LibstorageNG/SelectHardDisksPage.pm

```diff
@@ -19,12 +19,18 @@ use testapi;
 use parent 'Installation::WizardPage';
 
 use constant {
-    SELECT_HARD_DISKS_PAGE => 'inst-select-disk-to-use-as-root'
+    SELECT_HARD_DISKS_PAGE     => 'select-hard-disks-one-selected',
+    SELECT_HANDLING_PARTITIONS => 'inst-select-disk-to-use-as-root'
 };
 
-sub press_next {
+sub skip_hard_disk_selection {
```

Comment:
Ok, I see that you call `skip_handling_partitions` instead of `press_next` which became  `skip_hard_disk_selection`, so we are good to go. Cheers!

Reply:
I have the same doubt, because it says that is a WizardPage, so it means that include several screen or each screen should be a different class? or it is just a matter of how much I want to derive the model?